### PR TITLE
Fix code smells

### LINKED ISSUE #27: Follow-up on Sonar code smells
### Related software versions

all

### Description

With SonarCloud enabled, try to go through all code smells in a single PR so it does not pollute feature additions/fixes in other PRs.

### Use cases

As a maintainer, I would like to keep the code base simple and easy to work with.

### References

#26 - discussed here in the comments.

### PR DESCRIPTION
**Pull request by fopina**
Address code smells reported by [sonar](https://sonarcloud.io/project/issues?resolved=false&types=CODE_SMELL&id=surface-security_django-dkron) to help with #27

### CODE REVIEW

File: dkron/management/commands/run_dkron.py [hunk 1]
```diff
@@ -55,7 +55,6 @@ def download_dkron(self):
         return str(exe_path), str(bin_dir)
 
     def handle(self, *_, **options):
-        # TODO: check if there's any shutdown we should care before execv()
```
Comment (gsilvapt) [inline dkron/management/commands/run_dkron.py:58]: Should we move this to a new issue?

Comment (fopina) [reply dkron/management/commands/run_dkron.py:58]: I had that TODO as a generic comment of "is it ok to execv in a django command?" as there could be things/cleanup that should be performed.

But I'm not sure how to even start to check that so I was just plainly removing it 😄

Comment (fopina) [reply dkron/management/commands/run_dkron.py:58]: But we could track it in an issue actually, it doesn't hurt!

Comment (fopina) [reply dkron/management/commands/run_dkron.py:58]: https://github.com/surface-security/django-dkron/issues/32 created.

I'm not planning on hitting other smells for the time being, so if you're ok with this PR @gsilvapt, it's ready to merge

Comment (gsilvapt) [reply dkron/management/commands/run_dkron.py:58]: Thanks! This will help fix some of those smells and we can continue afterwards.

**Pull request merged**